### DISCUSSION TITLE
How to do vector fields under CF?

### DISCUSSION THREAD

**Discussion by twhitehead** [cf-convention/discuss#318] [Q&A about using CF] [answered]
### Question

Have been trying to figure out how to encode a vector (velocity) using the CF standard. Seems like there are at least three ways this could be done

* storing each component separably
* adding a dimension of length 3
* defining a custom compound type

but I can't figure out which would be compatible with the CF conversion. Seems that other have had this problem too. For example, here is someone asking on the ParaView  [mailing list](https://www.paraview.org/pipermail/paraview/2014-August/031913.html).

**Comment by sethmcg** [comment, marked as answer]
Option 1 is the standard way to store velocity fields.  If you search the standard name table for "velocity", you'll find lots of `something_something_x_velocity`, etc.

**Comment by ChrisBarker-NOAA** [comment]
NOTE: there have been multiple discussions in the past about having another way to more explicitly store vector data, but AFAIK, as @sethmcg said, your option (1) -- store each component individually, is the CF way.

**Comment by efisher008** [comment]
I believe this discussion has concluded and the original question has been answered, so I am marking this as closed.
